node/src/effect/announcements.rs

```diff
@@ -160,6 +160,19 @@ impl Display for UnexecutedBlockAnnouncement {
     }
 }
 
+#[derive(DataSize, Serialize, Debug)]
+pub(crate) struct StoredExecutedBlockAnnouncement(pub(crate) u64);
+
+impl Display for StoredExecutedBlockAnnouncement {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "announcement for stored executed block at height {}",
+            self.0,
+        )
+    }
+}
+
```

Comment:
I think we should look to use the existing `MetaBlock` functionality for this rather than creating a new announcement type.  When `ContractRuntime` has executed a block, after it has stored it, it calls `effect_builder.announce_meta_block(meta_block)`.  We can use this announcement to ensure the `BlockValidator` is also notified as part of the `MainReactor::handle_meta_block` flow.